.github/workflows/test.yml

```diff
@@ -14,6 +14,8 @@ env:
   MINT_LINK_PATH: .mint/bin
   MINT_PATH: .mint/lib
   SIMULATOR_DESTINATION: platform=iOS Simulator,name=iPhone 15 Pro,OS=17.0.1
+  RESULT_BUNDLE_PATH: build/AuthenticatorTests.xcresult
```

Comment:
```suggestion
  RESULT_BUNDLE_PATH: build/BitwardenTests.xcresult
```

Reply:
You're looking at drafts! I was mindlessly copying ... thanks for the catch.

Reply:
I completely missed that it was still a draft, sorry 😅